creation/interfaces-and-networks.md

```diff
@@ -548,6 +548,45 @@ the interface should be configured as follows.
 > **Note:** The network CIDR can be configured in the pod network
 > section using the `vmNetworkCIDR` attribute.
 
+#### masquerade - IPv6 support
+It is currently experimental, but `masquerade` mode can be used in IPv6 clusters
+(not dual stack - IPv6 only).
+
+As with the IPv4 `masquerade` mode, the VM can be contacted using the pod's IP
+address - which will be an IPv6 one. Outgoing traffic is also "NAT'ed" to the
+pod's IPv6 address.
+
+Unlike in IPv4, the configuration of the IP address, default route, and DNS is
+not automatic; it should be configured via cloud init, as shown below:
```

Comment:
or via the console.

Reply:
I think that's totally orthogonal to ipv6 or masquerade - the user can configure manually a static address on the VMs. No news here.

Reply:
I would at least change the "it should' to "it may".

Reply:
'Should' hints that this is the recommended way. I think that's a positive thing to have - a recommendation.

Let me know if you really want this done @AlonaKaplan , I don't have more arguments than this.

Reply:
It's not a big deal. I would still change it to "may":) There are vms with no cloud init and it's totally fine to configure the ips via the console.